NomaiVR/Helpers/MaterialHelper.cs

```diff
@@ -68,5 +68,20 @@ public static void DisableRenderersWithShaderInChildren(GameObject parent, Shade
                 }
             }
         }
+
+        public static void ApplyCanvasOpacity(Canvas canvas, float alpha)
+        {
+            CanvasGroup opacityGroup = canvas.GetComponent<CanvasGroup>();
+            if (opacityGroup == null)
+                opacityGroup = canvas.gameObject.AddComponent<CanvasGroup>();
+            opacityGroup.alpha = alpha;
+
+            foreach (Renderer renderer in canvas.GetComponentsInChildren<Renderer>(true))
+            {
+                var uiColor = renderer.material.color;
+                uiColor.a = alpha; //Squared for more drastic changes
```

Comment:
A bit confusing that you left this comment here when the actual squaring ks done elsewhere. Better to ether move the comment or move where the squaring happens.